pkg/compute/manifest/manifest.go

```diff
@@ -189,11 +189,20 @@ type File struct {
 	Authors         []string `toml:"authors"`
 	Language        string   `toml:"language"`
 	ServiceID       string   `toml:"service_id"`
+	Testing         Testing  `toml:"testing"`
```

Comment:
5. We update the manifest `File` data structure to reflect there is now a new `[testing]` section (as per the [`fastly.toml`](https://developer.fastly.com/reference/fastly-toml/) specification).